cheatsheets_draft/Secrets_Management_CheatSheet.md

```diff
@@ -254,179 +265,235 @@ For GCP, the recommended service is [Secret Manager](https://cloud.google.com/se
 
 Permissions are granted at the secret level.
 
-Check out the [Secret Manager best practices](https://cloud.google.com/secret-manager/docs/best-practices) for more information.
+Check out the [Secret Manager best practices](https://cloud.google.com/secret-manager/docs/best-practices).
 
 #### 4.1.3 Azure
 
 For Azure, the recommended service is [Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/).
 
-Contrary to other clouds, permisssions are granted at the _**Key Vault**_ level. This means secrets for separate workloads and separate sensitivity levels should be in separated Key Vaults accordingly.
+Contrary to other clouds, permissions are granted at the _**Key Vault**_ level. This means secrets for separate workloads and separate sensitivity levels should be in separated Key Vaults accordingly.
 
-Check out the [Key Vault best practices](https://docs.microsoft.com/en-us/azure/key-vault/general/best-practices) for more information.
+Check out the [Key Vault best practices](https://docs.microsoft.com/en-us/azure/key-vault/general/best-practices).
 
 ### 4.2 Envelope & client-side encryption
 
-There are various considereations when it comes to secrets management in the cloud. Two we want to deal with here, is how the secret is encrypted and how the keys for that encryption can be managed in the cloud.
+This section will describe how a secret is encrypted and how you can manage the keys for that encryption in the cloud.
 
 #### 4.2.1 Client-side encryption versus server-side encryption
 
-Server-side encryption of secrets ensures that the cloud-provider takes care of the encryption of the secret at storage. This means that the secret is safeguarded against compromise while being at rest. This often does not require any additional work, other than selecting the key to encrypt it with (See section 4.2.2). However: when the secret is submitted to another service to consume the secret, it will no longer be encrypted, as it is decrypted before submission to the intended service or human user where it sould be shared with.
+Server-side encryption of secrets ensures that the cloud provider takes care of the encryption of the secret at storage. The secret is then safeguarded against compromise while at rest. Encryption at rest often does not require additional work other than selecting the key to encrypt it with (See section 4.2.2). However: when you submit the secret to another service, it will no longer be encrypted. It is decrypted before sharing with the intended service or human user.
```

Comment:
```suggestion
Server-side encryption of secrets ensures that the cloud provider takes care of the encryption of the secret at storage. The secret is then safeguarded against compromise while at rest. Server-side encryption at rest often does not require additional work other than selecting the key to encrypt it with (See section 4.2.2). However: when you submit the secret to another service, it will no longer be encrypted. It is decrypted before sharing with the intended service or human user.
```